src/shared/utils/insertPreambleInScript.ts

```diff
@@ -26,7 +26,7 @@ export const insertPreambleInScript = async (
 
   const ast = resp.data.ast as Package
 
-  const imports: ImportDeclaration[] = get(ast, 'files.0.imports', [])
+  const imports: ImportDeclaration[] = ast?.files[0]?.imports ?? []
```

Comment:
`ast.files.0.imports` comes back as `null`. `_.get` returns the default value only if the value is `undefined`. Since it was null, this was failing and `imports` was being set to `null` before being mapped over on line 32. This fix is equivalent to:

```ts
get(ast, 'files.0.imports') || []
```